Add documentation for thumbnails regenerate command

PrestaEdit opened the pull request:
Introduces a markdown file detailing the usage of the `prestashop:thumbnails:regenerate` console command, including arguments, options, and example invocations for regenerating image thumbnails in PrestaShop.

<!-----------------------------------------------------------------------------
Thank you for contributing to the PrestaShop developer documentation! 

Please take the time to edit the "Answers" rows below with the necessary information.

Check out our contribution guidelines on how to contribute:
https://devdocs.prestashop-project.org/9/contribute/documentation/how/
------------------------------------------------------------------------------>

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.x
| Description?      | Documentation for https://github.com/PrestaShop/PrestaShop/pull/39478

<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->

PrestaEdit committed:
Add documentation for thumbnails regenerate command

Introduces a markdown file detailing the usage of the `prestashop:thumbnails:regenerate` console command, including arguments, options, and example invocations for regenerating image thumbnails in PrestaShop.

Touxten commented:
Well spotted! 

But for the moment, all new commands are in dev, so 9.1.x. :s 

We can backport the commad to 9.0.x. cc @kpodemski

Touxten reviewed (approved):
ok for me i think we can cherry pick this one 🤟

PrestaEdit commented:
There is no 9.1 branch for the doc. We could surely add the "since version" somewhere, maybe :-/

PrestaEdit committed:
Document minimum version for thumbnail regeneration

Added a note specifying that the prestashop:thumbnails:regenerate command is available since version 9.1.x.

kpodemski reviewed (approved):
thank you @PrestaEdit ! 💪🏻
